content/blog/2018-06-06-2018-unconf-security-recap.md

```diff
@@ -0,0 +1,53 @@
+
+Unconf projects x: 
+
+---
+slug: "unconf_recap_x"
```

Comment:
slug: unconf_recap_2

Reply:
@karthik this is recap 2 so

slug: "unconf_recap_2"
title: "Unconf18 projects 2: middlechild, defender, ropsec, keybase"